charts/prometheus/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: prometheus
-appVersion: v2.43.1
+appVersion: v2.43.2
```

Comment:
Please, raise `version`, not `appVersion`. As a new feature is being introduced, the minor version should go up.

Reply:
apologies for that